doc/tools/docgram/orderedGrammar

```diff
@@ -89,8 +89,8 @@ term_explicit: [
 ]
 
 primitive_notations: [
-| number
 | string
+| number
```

Comment:
```suggestion
| number
| string
```

Better to keep the existing order here with number first since the doc discusses number and string in that order.  Then please run `make doc_gram_rsts` to restore that order in the rst file.  Doc_grammar is unable to preserve the existing order for certain changes to the grammar (a defect in the mind-reading part of the code).